phc-rails/app/controllers/accounts_controller.rb

```diff
@@ -1,3 +1,26 @@
 class AccountsController < ApplicationController
+  respond_to :json
+  skip_before_action :verify_authenticity_token
+#  before_action :verify_security_token
 
+  def duplicates
+    # TEMP AUTH TOKEN FOR DEMOS
+    auth_token = request.params[:auth_token]
+    cursor = request.params[:cursor]
+    count = request.params[:count]
+    cursor = cursor.nil? ? 0 : cursor.to_i
+    count = count.nil? ? 20 : count.to_i
+    if auth_token != ENV['AUTH_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.